package.json

```diff
@@ -34,17 +34,21 @@
     "test": "jest",
     "test:coverage": "jest; open ./coverage/lcov-report/index.html",
     "check": "tsc --project ./tsconfig.check.json && pnpm lint",
-    "postinstall": "pnpm i rimraf && pnpm build"
+    "postinstall": "pnpm i rimraf && pnpm build",
+    "generate-types-from-schema": "cd src/export/digitalPlanning/schema/ && json2ts schema.json > types.d.ts"
```

Comment:
I did consider making this a script in the subfolder to avoid polluting this here, but it seemed pretty quick / pragmatic.